Sources/SwiftRoaring/RoaringBitmap.swift

```diff
@@ -961,16 +962,19 @@ public final class RoaringBitmap: Sequence, Equatable, CustomStringConvertible,
     /// Structure used to iterate through values in a roaring bitmap
     ///
     public struct RoaringBitmapIterator: IteratorProtocol {
-        private var i: UnsafeMutablePointer<roaring_uint32_iterator_t>
+        @usableFromInline
+        internal var i: roaring_uint32_iterator_t
 
-        init(ptr: UnsafeMutablePointer<roaring_bitmap_t>) {
-            self.i = croaring.roaring_create_iterator(ptr)
+        @inlinable @inline(__always)
+        init(ptr: UnsafePointer<roaring_bitmap_t>) {
+            self.i = croaring.roaring_create_iterator_value(ptr)
```

Comment:
Can't you just do  the equivalent of `roaring_init_iterator(ptr, &i)` (C syntax)?  Surely, we can pass a pointer to `i` from Swift to C?